config/policy/201-clusterrole.yaml

```diff
@@ -61,6 +61,13 @@ rules:
   verbs:
     - get
 
+- apiGroups: [“”]
```

Comment:
This looks unrelated to the renaming. Is it here intentionally?

Reply:
It is intentionally. During debugging I found the controller needs to have Event permission. So piggybacking here :D

Reply:
For changes of this size (almost three hundred files...), I prefer to not piggy back things like this. 

If you still want to, make sure to mention it in the PR's description. My reasoning is that if we need to roll this back (probably not for this particular PR, but somewhat common for large renaming changes), we don't want a functionality regression at the same time as a large name change. And based on the current description, I wouldn't expect any functionality changes, just name changes. So someone might do a straight rollback and not realize the problem.

Reply:
Ack. Description updated.